tests/Microsoft.Fx.Portability.MetadataReader.Tests/ManagedMetadataReaderTests.cs

```diff
@@ -17,7 +16,7 @@
 
 namespace Microsoft.Fx.Portability.MetadataReader.Tests
 {
-    public class ManagedMetadataReaderTests
+    public partial class ManagedMetadataReaderTests
```

Comment:
Why the change to partial? I couldn't find another partial class matching this.

Reply:
Already fixed :) was auto-added during a refactor